examples/testing-network-shim-app/cypress.env.json

```diff
@@ -2,5 +2,7 @@
     "dhis2_base_url": "http://localhost:1337",
     "dhis2_username": "admin",
     "dhis2_password": "district",
-    "dhis2_datatest_prefix": "testing-network-shim-app"
+    "dhis2_datatest_prefix": "testing-network-shim-app",
+    "dhis2_api_version": "36",
+    "network": "live"
```

Comment:
`network` defaults to `live`, but added it since it's an example env config.

The api version can be set to a default as well for less verbose `--env` flags. We could generate different `cypress.env.json` files for different environments with different versions to run against too, if we don't add the `cypress.env.json` to Git.